File: tops-work/SE/Assignment/Module-2(Fundamentals-of-Programming)/Looping-conditional-statements/que9.c
/*
Que9. Write a program make a summation of given number (E.g., 1523 Ans: 11)
*/

#include <stdio.h>
#include <stdlib.h>
int main()
{
    int real_number,number,digit[20],i = 0,sum = 0;
    printf("Enter any number\n");
    scanf("%d",&real_number);
    number = abs(real_number);
    while(number > 0)
    {
        digit[i] = number % 10;
        number = number / 10;
        i++;
    }
    for(int j = 0; j <= i; j++)
    {
       sum += digit[j];
    }
    printf("Sum of %d number is %d",real_number,sum);
   
    return 0;
}

File: tops-work/PYTHON/Assignment/Fundamentals-of-Python/que9.py
"""
Que9. Write a Python program to sum of three given integers. However, if
two values are equal sum will be zero. 
"""

num1 = int(input("Enter 1st number: "))
num2 = int(input("Enter 2nd number: "))
num3 = int(input("Enter 3rd number: "))
if num1 == num2 or num2 == num3 or num3 == num1:
    sum = 0
else:
    sum = num1 + num2 + num3
print("Sum of three integer numbers: ",sum)